### ISSUE TITLE
video -90 chrome and dont play safari

### ISSUE THREAD

**Issue by brunokunace** [aframevr/aframe#4422]
**Description:**

- A-Frame Version: 1.0.3
- Platform / Device: mac os catalina
- Reproducible Code Snippet or URL: https://codepen.io/brun00/pen/gObqByZ

Hi, when i get a video from android/ios camera and put on a-video, have 2 issues
1) on chrome rotate -90deg
2) on safari play only sound, image freeze (but not rotate)

on codepen have a video example
if get another video (not for any camera directly) works fine

Have any way to solve this fast? Thanks!

**Comment by brunokunace** [comment]
No one with this same issue?

**Comment by maxime-memtell** [comment]
Having the same issue right now, @brunokunace did you find a fix?

In my case it works as expected on Android and iPhone 6+ but not on iPhone 8+. I'm a bit confused. Looks like it comes from specific rotations or something...

**Comment by brunokunace** [comment]
@maxime-memtell no, i dont have any fix for that. And yes, its because a rotation script in the specific version of phones

**Comment by maxime-memtell** [comment]
@brunokunace thanks for your answer. Later in the day I debugged my video not showing, it was due to other issues (crossorigin="anonymous" on video tag and long loading times).

I also could fix the rotation but it made no sense. My a-box entity had rotation="45 180 90" as values and I changed it to rotation="45 180 0". This solved it on the iPhone 8 and didn't have any impact on the iPhone 6 and other android phone I had available... Weird fix uh
